docs/topics_5/web-openshift-troubleshoot.adoc

```diff
@@ -135,7 +135,7 @@ This error can be caused by a number of sources, with the following being the mo
 
 Attempt the deployment again, and view the logs and events of the non deployment Pods while they are being created. These messages will provide context to the underlying errors resulting in the deployment Pod timeouts.
 
-* To address the first issue reported, where the OpenShift instance is out of resources, follow the instructions in link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-resource-levels.html[Analyzing Cluster Capacity] to determine the cluster capacity. Once the capacity has increased, or there are fewer jobs executing, attempt the deployment once again.
+* To address the first issue reported, where the OpenShift instance is out of resources, follow the instructions in link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-resource-levels.html[Analyzing Cluster Capacity] in the OpenShift documentation to determine the cluster capacity. Once the capacity has increased, or there are fewer jobs executing, attempt the deployment once again.
```

Comment:
Remove "To address the first issue reported". That's rather lame. 

You could rewrite, "If the OpenShift instance runs out of resources, link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-resource-levels.html[analyze your cluster's capacity]." This is an inline link. It does not require mentioning the section name or documentation. I think it looks better than "See the x section in the y documentation". It's standard for OpenShift docs.

The last sentence is really obscure. Is the capacity going to increase by itself? What makes fewer jobs execute? In tech writing, the actor must be explicit. This is all passive voice and the actor is not clear. If you can fix this, please do. Don't hold up the job for this but it should be fixed by someone eventually.